inflammation/models.py

```diff
@@ -31,3 +31,14 @@ def daily_max(data):
 def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
+
+
+def s_dev(data):
+    """Computes and returns standard deviation for data."""
+    number = np.mean(data, axis=0)
+    devs = []
+    for entry in data:
+        devs.append((entry - number) * (entry - number))
+
+    s_dev2 = sum(devs) / len(data)
```

Comment:
Perhaps we can use [`np.std`](https://numpy.org/doc/stable/reference/generated/numpy.std.html) instead

Reply:
Duh! :doughnut:  Agreed.